seahorn/trusty/user/base/lib/tipc/proof/tipc_connect/main.c

```diff
@@ -0,0 +1,13 @@
+#include <seahorn/seahorn.h>
+#include <lib/tipc/tipc.h>
+#include <trusty_ipc.h>
+
+int main(void) {
+  int rc;
+  handle_t h = INVALID_IPC_HANDLE;
+  char path[64];
```

Comment:
unused?

Reply:
This is the example Xiang set for me. The goal is to teach me how to write tests :)